jobs/build/release/Jenkinsfile

```diff
@@ -48,7 +48,7 @@ node {
                     ],
                     [
                         name: 'PREVIOUS',
-                        description: 'Optional: Tag(s) (comma separated) of last 10 releases this can upgrade FROM',
+                        description: 'List all previous releases to perform upgrade tests\nhttps://openshift-release.svc.ci.openshift.org/',
```

Comment:
Actually it's supposed to be basically what _has already_ passed upgrade tests (from previous release to nightly).

Reply:
As we discussed, better to just point it to the docs ;D